src/routes/_components/status/StatusMediaAttachments.html

```diff
@@ -79,7 +91,9 @@
   .status-sensitive-media-container.status-sensitive-media-hidden {
     width: 100%;
     margin: 10px auto;
-    height: 200px;
+  }
+  .status-sensitive-media-container.status-sensitive-media-hidden:not(.grouped-images) {
```

Comment:
I tend not to use `:not` selectors (again, for perf). We can use a `.not-grouped-images` class here if we need to